akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -272,12 +274,6 @@ private[stream] object TcpConnectionStage {
 
       override def onUpstreamFailure(ex: Throwable): Unit = {
         if (connection != null) {
-          if (interpreter.log.isDebugEnabled) {
```

Comment:
So help my address my problem in some other way then. "I am Akka HTTP I WILL log this error in a MUCH BETTER WAY than anyone in this class is able to. How can I do that without scaring the shit out of people with stacktraces here.

I need either a flag for this or a way to customise in some way.